tensorboard/plugins/hparams/download_data.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Classes and functions for handling the DownloadData API call."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+from tensorboard.plugins.hparams import error
```

Comment:
nit: Please [group and order imports per the style guide][1], and [only
import modules][2] (`import six`, not `from six import StringIO`).

[1]: https://google.github.io/styleguide/pyguide.html#313-imports-formatting
[2]: https://google.github.io/styleguide/pyguide.html#22-imports